.env.example

```diff
@@ -23,8 +23,10 @@ MAIL_SENDER='some-email-account@example.com'
 # MAIL_SMTP_PASSWORD='XXX'
 # MAIL_SMTP_TLS='true'
 
-# Controls URLs that won't be downloaded and re-hosted when importing versions
+# Controls URLs that won't be downloaded and re-hosted when importing versions. These will be exposed directly to clients via redirects rather than being proxied by web-monitoring-db.
 ALLOWED_ARCHIVE_HOSTS='https://edgi-web-monitoring-db.s3.amazonaws.com/ https://edgi-wm-versionista.s3.amazonaws.com/ https://edgi-wm-versionista.s3-us-west-2.amazonaws.com/ https://s3-us-west-2.amazonaws.com/edgi-wm-versionista/ https://edgi-versionista-archive.s3.amazonaws.com/ https://edgi-versionista-archive.s3-us-west-2.amazonaws.com/ https://s3-us-west-2.amazonaws.com/edgi-versionista-archive/'
+# OPTIONAL: Additional URLs that will be exposed directly to clients via redirects rather than being proxied by web-monitoring-db.
+#PUBLIC_ARCHIVE_HOSTS='http://static.example.com http://files.example.com'
```

Comment:
I know I suggested this direction, but the more I think about it, the more complex it feels. We already reliably confuse people with `ALLOWED_ARCHIVE_HOSTS` (see #249) and this just adds to the complexity there and adds a lot to `Archiver`. I’m wondering if a better model might be to indicate whether the storage is public. So `FileStorage::LocalFile` would always be private, but `FileStorage::S3` might check `@acl`.

Thoughts? If you disagree, go ahead and stick with this. If you do, though, I think it needs more explanation to clarify that it’s essentially *additive* on top of `ALLOWED_ARCHIVE_HOSTS`.

Reply:
Yeah, we definitely have plenty of complexity that leaks into configuration. Let me think on this.

Reply:
`PUBLIC_ARCHIVE_HOSTS` is no more.